### PR TITLE
University Physics recipe

### PR DESCRIPTION
**Pull request by kswanson33**

### CODE REVIEW

File: books/u-physics/bake [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+  BakeMathInParagraph.v1(book: book)
+
+  # Fix titles in chapter review composite pages
+  # TODO: remove this
+  book.chapters('.os-eoc.os-chapter-review-container').composite_pages.each do |page|
+    title = page.search('[data-type="document-title"]').first
+    title[:'data-type'] = 'title'
+    title.name = 'h3'
+    page.metadatas.first.search('[data-type="document-title"]').first.replace_children(with: 'Chapter Review')
+  end
```
Comment (kswanson33) [inline books/u-physics/bake:101]: will be addressed by openstax/kitchen#207

File: books/u-physics/bake [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+after = Kitchen::BookRecipe.new(book_short_name: :u_physics) do |doc|
+  include Kitchen::Directions
+
+  book = doc.book
+
+  # remove w3 from toc
+  # TODO: remove this
+  book.search('nav#toc').first.search('[xmlns="http://www.w3.org/1999/xhtml"]').each do |el|
+    el.remove_attribute('xmlns')
+  end
```
Comment (kswanson33) [inline books/u-physics/bake:122]: will be addressed by openstax/kitchen#209

**Review by kswanson33 — COMMENTED** [review]
Specs should currently fail because of the kitchen version. Waiting on a few more changes to be added to Kitchen, but this should be the final or near-final form of this recipe.

File: books/u-physics/bake [hunk 3]
```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+require 'bundler/inline'
+
+gemfile do
+  gem 'openstax_kitchen', ENV['USE_LOCAL_KITCHEN'] ? { path: '/code/kitchen' } : '2.0.0'
```
Comment (jpslav) [inline books/u-physics/bake:8]: update this to a more recent release

File: books/u-physics/bake [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+
+  BakePreface.v1(book: book)
+  BakeChapterIntroductions.v1(book: book)
+  BakeChapterTitle.v1(book: book)
+  book.chapters.each do |chapter|
+    BakePageAbstracts.v1(chapter: chapter)
+    BakeNonIntroductionPages.v1(chapter: chapter)
+  end
+
+  solutions_container = BookAnswerKeyContainer.v1(book: book)
```
Comment (jpslav) [inline books/u-physics/bake:28]: Can this move down after the next three notes steps to be closer to where it is used in the following loop?

File: books/u-physics/bake (hunk 2 above)
Comment (jpslav) [reply books/u-physics/bake:122]: this has been merged

**Review by jpslav — CHANGES_REQUESTED** [review]

File: books/u-physics/bake (hunk 3 above)
Comment (kswanson33) [reply books/u-physics/bake:8]: sure! there isn't a release yet that has _everything_ in it but i'll put the most recent

File: books/u-physics/bake [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+    BakeChapterGlossary.v1(chapter: chapter, metadata_source: metadata, append_to: eoc_wrapper)
+    BakeChapterKeyEquations.v1(chapter: chapter, metadata_source: metadata, append_to: eoc_wrapper)
+    BakeChapterKeyConcepts.v1(chapter: chapter, metadata_source: metadata, append_to: eoc_wrapper)
+    # EoC sections with exercises
+    MoveExercisesToEOC.v2(chapter: chapter, metadata_source: metadata, append_to: eoc_wrapper, klass: 'review-conceptual-questions')
+    MoveExercisesToEOC.v2(chapter: chapter, metadata_source: metadata, append_to: eoc_wrapper, klass: 'review-problems')
+    MoveExercisesToEOC.v1(chapter: chapter, metadata_source: metadata, append_to: eoc_wrapper, klass: 'review-additional-problems')
+    MoveExercisesToEOC.v1(chapter: chapter, metadata_source: metadata, append_to: eoc_wrapper, klass: 'review-challenge')
+    # In-place bake exercises & solutions
+    chapter.search('section.review-conceptual-questions, section.review-problems, section.review-additional-problems, section.review-challenge').exercises.each do |exercise|
```
Comment (jpslav) [inline books/u-physics/bake:48]: line length here and elsewhere.  do we use rubocop here?

**Review by jpslav — CHANGES_REQUESTED** [review]

**Review by jpslav — APPROVED** [review]

**Pull request merged**